reproman/tests/skip.py

```diff
@@ -56,6 +56,7 @@ def windows():
 # function to CONDITION_FNS.
 
 
+# TODO(asmacdo) maybe should make new `no_debian`
```

Comment:
may be.... it is a bit ambigious on the word though, might be `not_on_debian` since there could be `debian` available via WSL on windows etc ;)

Reply:
what was your idea/use-case here more specifically?

Reply:
2 tests were not skipped on fedora:
- https://github.com/ReproNim/reproman/pull/584/files#diff-2ed961bebca8aef436f7a673805596bae4c80aeed8836eb3593d87a0123db3e7R129
 - https://github.com/ReproNim/reproman/pull/584/files#diff-2ed961bebca8aef436f7a673805596bae4c80aeed8836eb3593d87a0123db3e7R140

I put this todo to make sure that skipif no apt is appropriate for those 2, or if it should just be not deb.

Reply:
apt-cache is sufficieent for now